docs/user-guide/isis/zoom.ipynb

```diff
@@ -43,168 +35,156 @@
    "outputs": [],
    "source": [
     "import scipp as sc\n",
-    "import sciline\n",
     "from ess import sans\n",
     "from ess import isissans as isis\n",
     "from ess.sans.types import *"
    ]
   },
   {
    "cell_type": "markdown",
-   "id": "64783c51",
+   "id": "d1bb87a9",
    "metadata": {},
    "source": [
-    "### Setup input files"
+    "## Create and configure the workflow\n",
+    "\n",
+    "We begin by creating the Zoom workflow object (this is a [sciline.Pipeline](https://scipp.github.io/sciline/generated/classes/sciline.Pipeline.html) which can be consulted for advanced usage).\n",
+    "The Zoom workflow uses Mantid to load files.\n",
+    "This tutorial comes with files that do not require Mantid, so we use slightly modified a workflow that does not require Mantid:"
    ]
   },
   {
    "cell_type": "code",
    "execution_count": null,
-   "id": "9e7cfc82",
+   "id": "955bbc91",
    "metadata": {},
    "outputs": [],
    "source": [
-    "params = isis.default_parameters()\n",
-    "params.update(\n",
-    "    {\n",
-    "        DirectBeamFilename: 'Direct_Zoom_4m_8mm_100522.txt',\n",
-    "        isis.CalibrationFilename: '192tubeCalibration_11-02-2019_r5_10lines.nxs',\n",
-    "        Filename[SampleRun]: 'ZOOM00034786.nxs',\n",
-    "        Filename[EmptyBeamRun]: 'ZOOM00034787.nxs',\n",
-    "        isis.SampleOffset: sc.vector([0.0, 0.0, 0.11], unit='m'),\n",
-    "        isis.DetectorBankOffset: sc.vector([0.0, 0.0, 0.5], unit='m'),\n",
-    "    }\n",
-    ")\n",
-    "masks = [\n",
-    "    'andru_test.xml',\n",
-    "    'left_beg_18_2.xml',\n",
-    "    'right_beg_18_2.xml',\n",
-    "    'small_bs_232.xml',\n",
-    "    'small_BS_31032023.xml',\n",
-    "    'tube_1120_bottom.xml',\n",
-    "    'tubes_beg_18_2.xml',\n",
-    "]"
+    "workflow = isis.zoom.ZoomTutorialWorkflow()\n",
+    "# For real data use:\n",
+    "# workflow = isis.zoom.ZoomWorkflow()"
```

Comment:
I see nothing wrong with the repetition, maybe we will find it useful in the name. Also, otherwise you have multiple identically named functions or classes in different submodules of the same package.